.github/quarkus-github-bot.yml

```diff
@@ -226,6 +226,10 @@ triage:
         - extensions/security/
         - extensions/elytron
         - integration-tests/elytron
+    - id: bearer-token
+      labels: [area/security]
```

Comment:
Ideally it should be both `area/oidc` and `area/security`, `area/oidc` should cover most of these issues, but `area/security` can also cover smallrye-jwt, I was about to propose an update but not sure, if it is `,` or ` ` which separates multiple values :-)